wdl/GenotypeDepthPart2.wdl

```diff
@@ -202,3 +211,39 @@ task IntegrateDepthGq {
     maxRetries: select_first([runtime_attr.max_retries, default_attr.max_retries])
   }
 }
+
+task MergeRegenoCoverageMedians {
+  input {
+    Array[File] regeno_coverage_medians_array
+    String batch
+    String sv_base_mini_docker
+    RuntimeAttr? runtime_attr_override
+  }
+
+  RuntimeAttr default_attr = object {
+    cpu_cores: 1, 
+    mem_gb: 3.75,
+    disk_gb: 10,
+    boot_disk_gb: 10,
+    preemptible_tries: 3,
+    max_retries: 1
+  }
+  RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
+
+  output {
+    File regeno_coverage_medians = "~{batch}.regeno.coverage_medians_merged.bed"
+  }
+  command <<<
+    cat ~{sep=' ' regeno_coverage_medians_array} | fgrep -v "start" > ~{batch}.regeno.coverage_medians_merged.bed
```

Comment:
I'd assume `-v start` is to filter out a header line. Can we search for the full header line for the off chance that `start` appears in a record?

Reply:
Done (grep first 4 columns of header, excluding sample IDs), and tested with test_large batch to make sure it ran properly through Module04.GenotypeDepthPart2 and the regeno_coverage_medians output was identical